dspace-api/src/main/java/org/dspace/discovery/IndexingService.java

```diff
@@ -51,14 +51,9 @@ void reIndexContent(Context context, IndexableObject dso)
 
     void updateIndex(Context context, boolean force);
 
-    void updateIndex(Context context, boolean force, int type);
```

Comment:
Would it be possible to keep the updateIndex methods with type (but type as string)?
I think it would imply getting the IndexFactory for the given type, and only processing that factory.

This would allow for reindexing e.g. only the workspace items

Reply:
This method was re-added as requested.